### ISSUE TITLE
How could we integrate with ripcord?

### ISSUE THREAD

**Issue by davidperezgar** [gggeek/polyfill-xmlrpc#3]
We are using the library ripcord to connect to an ERP. How could we use both of them?

**Comment by gggeek** [comment]
Hello.
It should be seamless usage, ie. nothing to do other than installing both ripcord and polyfill-xmlrpc via Composer.
Or did you try that out already and found out that something is not working as expected?

**Comment by gggeek** [comment]
Ping - any updates?

**Comment by gggeek** [comment]
Ok, so I have added examples of using this with ripcord, both client and server:

https://github.com/gggeek/polyfill-xmlrpc/tree/master/demo/client/ripcord
https://github.com/gggeek/polyfill-xmlrpc/tree/master/demo/server/ripcord

Btw, I have used the version of ripcord from darkaonline - as it seems that there are many versions of ripcord floating around Packagist, and the darkaonline one is the one with most starts/downloads and somewhat recent commits.
However, it took me all of 5 minutes of playing around to find broken stuff in it.
I would recommend moving away from ripcord if possible.
F.e. https://github.com/gggeek/phpxmlrpc-extras has a self-documenting server which could easily replace the ripcord one, and the `proxy.php` client from the phpxmlrpc demo folder might replace the ripcord client...
